CHANGELOG.md

```diff
@@ -57,7 +57,7 @@ The types of changes are:
 - Fix an error where a connector response value of None causes a DSR failure due to a missing value [#4483](https://github.com/ethyca/fides/pull/4483)
 - Fixed system name being non-editable when locked for GVL [#4475](https://github.com/ethyca/fides/pull/4475)
 - Fixed a bug with "null" values for retention period field on data uses [#4487](https://github.com/ethyca/fides/pull/4487)
-- Fixed an issue with the Iterate connector returning At least one param_value references an invalid field for the 'update' request of user [#4514]
+- Fixed an issue with the Iterate connector returning At least one param_value references an invalid field for the 'update' request of user [#4528]
```

Comment:
This needs to be moved to the `Unreleased` section, and make sure to add the rest of the link
```
[#4528](https://github.com/ethyca/fides/pull/4528)
```

Reply:
Thanks! Fixed this.